src/unix/linux_like/mod.rs

```diff
@@ -616,6 +616,8 @@ pub const MS_LAZYTIME: ::c_ulong = 0x2000000;
 pub const MS_ACTIVE: ::c_ulong = 0x40000000;
 pub const MS_MGC_VAL: ::c_ulong = 0xc0ed0000;
 pub const MS_MGC_MSK: ::c_ulong = 0xffff0000;
+pub const OPEN_TREE_CLONE: ::c_uint = 0x01;
+pub const OPEN_TREE_CLOEXEC: ::c_uint = O_CLOEXEC as ::c_uint;
```

Comment:
These have been added in linux_like mod but I guess emscripten doesn't have them?

Reply:
I'm honestly not very familiar with emscripten, but yes it doesn't seem to have these (also, no `open_tree()` at all).
At the same time, I think those are not defined in any C lib either, only in the Linux kernel uapi headers.

Is this a concern? Should I move these constant to `android/` and `linux/` instead?

Reply:
> I'm honestly not very familiar with emscripten, but yes it doesn't seem to have these (also, no `open_tree()` at all). At the same time, I think those are not defined in any C lib either, only in the Linux kernel uapi headers.

If I checked correctly, it doesn't have yet:
https://github.com/emscripten-core/emscripten/blob/6a93f7db16cd8e5214b4c3c08a361ccab9ffc67e/system/lib/libc/musl/include/sys/mount.h

> Is this a concern? Should I move these constant to `android/` and `linux/` instead?

It's not a big deal as the build passes fine anyway. But I'd prefer to add items only if a target has it, because it would produce a compile error instead of a weird/unclear behavior due to a wrong const/flag usage, I feel the former is more helpful for end users.

Reply:
Ack. I've moved them to Android plus Linux only then.